Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM ubuntu:16.04
 
 RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
     apt-get -yqq install \
+    git \
```

Comment:
?

Reply:
Apparently git isn't installed by default in that image. Tests were failing in Docker because the command was not found.